web/libs/editor/src/components/AnnotationsCarousel/AnnotationsCarousel.scss

```diff
@@ -7,6 +7,27 @@
   min-width: 0;
   flex: 1;
 
+  &::before {
+    content: '';
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 8px;
+    height: 100%;
+    display: block;
+    pointer-events: none;
+    opacity: 0;
+    background: linear-gradient(to right, rgba(var(--color-neutral-shadow-raw) / 16%), rgba(var(--color-neutral-shadow-raw) / 0%));
+    z-index: 1;
+    transition: all 200ms ease-out;
```

Comment:
we have some rendering performance degradations in some places because of transition added for `all`. here is only `opacity` changed, right? update it please to have a reduced scope